realworld/realworld.json

```diff
@@ -750,9 +750,9 @@
     "domain": "Justice",
     "tags": "legal, forms",
     "summary": "Ergonomic and accessible text system for the German justice system",
-    "description": "bk.text is an ergonomic and accessible text system for the German justice system, which supports the digital processing of court proceedings.",
+    "description": "bk.text is an ergonomic and accessible text system for the German justice, which supports the digital processing of judicial proceedings and business transactions in courts and public prosecutor’s offices.",
```

Comment:
There are some funny characters at the end of the description (see "prosecutor").